examples/map-variable.py

```diff
@@ -0,0 +1,40 @@
+"""
+Map-Variable
+============
+
+An example of how map-variable works.
+"""
+
+import numpy as np
+from matplotlib import pyplot as plt
+import WrightTools as wt
+
+# create original arrays
+x = np.linspace(1, 5, 11)[:, None]
+y = np.linspace(1, 5, 11)[None, :]
+arr = np.exp(-1 * (((x - 3) / .6) ** 2 + ((y - 3) / .6) ** 2))
+# create data object
+d = wt.data.Data(name="original")
+d.create_variable("x", values=x)
+d.create_variable("y", values=y)
+d.create_channel("z", arr)
+d.transform("x", "y")
+# create new data objects
+pointsx = np.linspace(x.min(), x.max(), 31)  # linear spacing
+pointsy = points = np.logspace(0, .7, 11)  # log spacing
+dx = d.map_variable("x", points=pointsx)  # just linear along x
```

Comment:
Do we wish to demonstrate just passing an int here?

Reply:
I am indifferent.